src/SFA.DAS.EAS.Web/Orchestrators/CommitmentsBaseOrchestrator.cs

```diff
@@ -95,11 +95,16 @@ public class CommitmentsBaseOrchestrator
             }
         }
 
-        protected async Task<List<ITrainingProgramme>> GetTrainingProgrammes()
+        protected async Task<List<ITrainingProgramme>> GetTrainingProgrammes(bool includeFrameworks)
```

Comment:
I personally would have added a default of true, so that existing consumers wouldn't have to be changed.